cmd/server/server.go

```diff
@@ -106,6 +106,9 @@ func (s *server) startService() common.Daemon {
 
 	params := service.BootstrapParams{}
 	params.Name = "cadence-" + s.name
+	if len(s.cfg.Log.LevelKey) == 0 {
+		s.cfg.Log.LevelKey = "level"
+	}
```

Comment:
Can you move this code to the `NewZapLogger`? 

also use `if s.cfg.Log.LevelKey == "" {` as our convention

Reply:
done